Xamarin.Essentials/TextToSpeech/TextToSpeech.macos.cs

```diff
@@ -51,7 +51,7 @@ internal static async Task PlatformSpeakAsync(string text, SpeechOptions options
 
             void TryCancel()
             {
-                ss.StopSpeaking(NSSpeechBoundary.hWord);
+                ss.StopSpeaking(NSSpeechBoundary.Word);
```

Comment:
This is a tricky one as this means anyone that hasn't updated to the latest version this may cause issues... @mattleibow ....

Reply:
Yeah. I was thinking that.

But, this only applies to those building this source code. As enums are constants, only the value is used in IL.

It should be OK.

Reply:
But, I think we should test to confirm my logic here. It should be a matter of just downloading this built assembly and viewing the IL.

I'll have a look to see.

Reply:
I'm setting up a VS2017 VM right now and I will check this.